package.json

```diff
@@ -15,36 +22,71 @@
     "@types/node": "^16.11.39",
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
+    "@types/react-redux": "^7.1.25",
+    "axios": "^1.3.5",
+    "bootstrap": "^5.2.3",
     "classnames": "^2.3.1",
-    "eslint-plugin-jest": "^27.0.1",
+    "eslint-plugin-jest": "^27.2.1",
     "formik": "^2.2.9",
-    "material-react-table": "^0.40.8",
+    "material-react-table": "^1.11.3",
+    "primeicons": "^6.0.1",
+    "primereact": "^9.4.0",
```

Comment:
`ramda` sounds interesting.  However, it's only being used for its `mergeDeepRight` function.  Maybe a lodash could provide it instead?